lib/whitespace.coffee

```diff
@@ -3,11 +3,12 @@
 module.exports =
 class Whitespace
   constructor: ->
```

Comment:
The `save-without-modifying` command is implemented by unsubscribing from editor events, issuing the save command, and re-registering the events. Split into two `CompositeDisposable`s so we can deregister the editor events without deregistering the command events.